<launch>
  <node name="action_client" pkg="example_action" type="action_client.py" output="screen"/>
  <node name="action_server" pkg="example_action" type="action_server.py" output="screen"/>
</launch>
